src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -177,6 +186,10 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
         }
     }
 
+    private void assignTopic(final EventType eventType) {
+        eventType.setTopic(uuidGenerator.randomUUID().toString());
```

Comment:
I believe for debugging purposes it would be better if topic name is prefixed with event-type name. What do you think?